pvlib/modelchain.py

```diff
@@ -267,6 +267,10 @@ def get_orientation(strategy, **kwargs):
 class ModelChainResult:
     _T = TypeVar('T')
     PerArray = Union[_T, Tuple[_T, ...]]
+    _singleton_tuples: bool = field(default=False)
+    _per_array_fields = {'total_irrad', 'aoi', 'aoi_modifier',
+                         'spectral_modifier', 'cell_temperature',
+                         'effective_irradiance', 'dc', 'diode_params'}
     """Type for fields that vary between arrays"""
```

Comment:
this doc string isn't part of the diff, but its location doesn't make much sense while reading it. should this be a comment instead and located above the `PerArray` line?

Reply:
Definitely in the wrong spot. Should be after the `PerArray` line so autodoc will pick it up in table of `ModelChainResult` attributes.